WebAPI.Domain/Queries/QueryParameters.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WebAPI.Domain.Queries
+{
+    /// <summary>
+    /// Holds query parameters get from URI
+    /// </summary>
+    public class QueryParameters
+    {
+        public int pageNumber { get; set; } = 1;
+        public int pageSize { get; set; } = 5;
+        public string sortBy { get; set; } = "Country";
+        public bool orderByDescending { get; set; } = false;
+        public string filterBy { get; set; } = "None";
```

Comment:
You can skip "None" and leave it empty. So if there is no filrer FilterBy prop will be null or empty.